app/controllers/YourSelfAssessmentChargesController.scala

```diff
@@ -66,13 +67,19 @@ class YourSelfAssessmentChargesController @Inject()(val authActions: AuthActions
 
       val hasOverdueCharges: Boolean = whatYouOweChargesList.chargesList.exists(_.isOverdue()(dateService))
       val hasAccruingInterestReviewAndReconcileCharges: Boolean = whatYouOweChargesList.chargesList.exists(_.isNotPaidAndNotOverduePoaReconciliationDebit()(dateService))
+      val earliestTaxYearAndAmount: (TaxYear, BigDecimal) =
```

Comment:
same here, fee free to create a new case class if you need to as tuples  use is generally discouraged; if this is something used within a function and not exposed outside of it, than this should be fine